### PR TITLE
Feature/now with auto

### PR DESCRIPTION
**Pull request by gnituy18**
This PR do : 

1. Add `auto (increase time)` condition to `lib/now` and `lib/isToday`.
2. Rewrite `utils` spec for better understanding.
3. Use `ms` as standard unit in time.

### CODE REVIEW

**Comment by rwu823** [comment]
Please also update README.md

File: .eslintrc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "@17media/17media",
+  "extends": "@17media/eslint-config-17media",
```
Comment (rwu823) [inline .eslintrc]: Why change this? It does't work?

File: src/utils/isAuto.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { isNode } from './env';
+
+const isAuto = () => (isNode
+  ? process.argv.indexOf('--auto') !== -1
+  : global.location.search.indexOf('auto=true') !== -1);
```
Comment (rwu823) [inline src/utils/isAuto.js]: It would be `true` if url is `?&abcdeauto=true`

File: .eslintrc (hunk 1 above)
Comment (gnituy18) [reply .eslintrc]: https://github.com/17media/style-guide#installation--eslint-setup
No, my linter didn't run correctly with `@17media/17media`

**Comment by rwu823** [comment]
LGTM

**Pull request merged**